exercises/all-your-base/all_your_base_test.cpp

```diff
@@ -0,0 +1,175 @@
+#include "all_your_base.h"
+
```

Comment:
Super-nitpick: I think all the problems don't have any whitespace in the top includes, so please remove this line for consistency.